wremnants/theory_tools.py

```diff
@@ -413,14 +416,18 @@ def moments_to_angular_coeffs(hist_moments_scales, cutoff=1e-5, sumW2=False):
 
     # e.g. from arxiv:1708.00008 eq. 2.13, note A_0 is NOT the const term!
     offsets = np.array([0., 4., 0., 0., 0., 0., 0., 0., 0.])
+    #print("vals = ", vals, "norm = ", norm_vals)
 
-    coeffs = vals / norm_vals + offsets
+    coeffs = vals / norm_vals# + offsets
```

Comment:
IS there some motivation for moving this?

Reply:
This will fix the stat uncertainty of A0. It will properly consider the variance or SumW2 as the root if we use 4 - 10 * cos^2 instead of applying offsets and scales in the later stage.